frontend/src/lib/lemon-ui/LemonDivider/LemonDivider.tsx

```diff
@@ -11,6 +12,10 @@ export interface LemonDividerProps {
     dashed?: boolean
     /** Adding a className will remove the default margin class names, allowing the greatest flexibility */
     className?: string
+    /* The position of title inside divider */
+    orientation?: 'left' | 'right' | 'center'
```

Comment:
I'm hesitant to add this prop given we so infrequently use the title (two cases in the entire app). I think we have two options here:
1) Make the title an absolutely positioned element that is wrapped alongside the divider in a container div
2) Do not allow the orientation to be set and always use the left aligned version (removing the need for a lot of the extra CSS above)